Color Replaced

Aakash2408 opened the pull request:
## Fixes
<!-- If PR doesn't fully resolve the issue, replace 'Fixes' below with 'Related to'. -->
<!-- If there is no issue being resolved, open one before creating this pull request. -->
Fixes #[#62 ] by @[@panchovm]

## Description
<!-- Concisely describe what the pull request does. -->
Color change from  DarkTurquoise with ForestGreen across all site.

## Technical details
<!-- Add any other information or technical details about the implementation; or delete the section entirely. -->
    rgb(34,139,34):This is the value of Forest Green






## Screenshots
<!-- Add screenshots to show the problem and the solution; or delete the section entirely. -->
![Screenshot from 2020-08-18 08-21-10](https://user-images.githubusercontent.com/47134609/90465152-635be980-e12c-11ea-8b50-4fef274e4657.png)

## Checklist
<!-- Replace  the [ ] with [x] to check the boxes. --> 

- [x] My pull request targets the *develop* branch of the repository.
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.

[best_practices]:https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53

kgodey commented on legal_db/static/styles.scss:
```diff
@@ -1,5 +1,6 @@
 $color-tomato: rgb(237, 89, 47);
 $color-dark-turquoise: rgb(5, 181, 218);
```
please remove this since it's no longer used

Aakash2408 replied:
ok i wil remove it

kgodey commented:
@Aakash2408 I switched the base branch to be `develop` since `develop` is not merged into `master` yet.

Please fix the merge conflicts by merging the latest `develop` into your branch.

Aakash2408 commented:
> @Aakash2408 I switched the base branch to be `develop` since `develop` is not merged into `master` yet.
> 
> Please fix the merge conflicts by merging the latest `develop` into your branch.
ok I will do it

Aakash2408 commented:
> @Aakash2408 I switched the base branch to be `develop` since `develop` is not merged into `master` yet.
> 
> Please fix the merge conflicts by merging the latest `develop` into your branch.

At that time I just switched  to develop branch, created my new branch and made changes into that and made a pull request to merge that into the main branch

Aakash2408 commented:
@kgodey  sorry there were major issues on my locals for rebasing and merge. So I opened another pull request kindly check #72 .
